tests/conftest.py

```diff
@@ -210,6 +209,7 @@ def test_server_with_valid_file(test_server, test_orm):
     file = test_orm.File.new_file(
         filename="example_file.txt",
         size=len(data),
+        # Leave as-is to test auto-selection of md5
```

Comment:
This is great to test this, but do we also test using the new xxh3 path? It looks like it's the new default in the hashing function, so I'm assuming other tests would do this, but wanted to confirm.

Reply:
Yes, all 'actual' uploads use the new xxh3-based checksums.